src/Totem/ClientApp/App.vue

```diff
@@ -352,7 +352,7 @@ export default {
         $('#ModifiedContract_ContractString')[0].value = this.modifiedContract;
         this.rows = parseContractArray(this.modifiedContract, 'contract-string-validation');
         this.isDescending = false;
-        if (updatedModel.rowId !== undefined) {
+        if (updatedModel.isNewModel && typeof setSaveButton === 'function') {
```

Comment:
the .isNewModel part is not correct; if you were editing a contract, and editing just a model that already existed, it would not set the save button and it should. I think it would probably be correct to just have `if (typeof setSaveButton === 'function')` as the entire condition, unless there's still a valid reason to care about rowId.